### PR TITLE
migrate urlsafe-base64 to be internal to web-push

### PR DESCRIPTION
**Pull request by hikarunoryoma**
Solves: https://github.com/web-push-libs/web-push/issues/785

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -43,9 +42,9 @@
     "eslint-config-airbnb": "19.0.4",
     "eslint-plugin-import": "2.26.0",
     "geckodriver": "3.2.0",
-    "nyc": "15.1.0",
```
Comment (marco-c) [inline package.json:46]: could you move this back?

**Comment by hikarunoryoma** [comment]
> LGTM, could you rebase?

Done! :)

**Comment by marco-c** [comment]
There seems to be one reference left to fix:
```
> node ./node_modules/eslint/bin/eslint --ignore-path .gitignore '.'


/home/runner/work/web-push/web-push/test/test-vapid-helper.js
Error:   18:26  error  'urlBase64' is not defined  no-undef

✖ 1 problem (1 error, 0 warnings)
```

**Comment by marco-c** [comment]
Unfortunately it looks like the tests are failing

**Comment by hikarunoryoma** [comment]
Thanks, I'll look to investigate this weekend and actually have JVM and run these test locally on my machine

**Comment by marco-c** [comment]
@hikarunoryoma still planning on working on this?

**Comment by haouarihk** [comment]
when is this gonna get merged.

Thanks for the hard work

**Comment by marco-c** [comment]
> when is this gonna get merged.
> 
> Thanks for the hard work

There are some failing tests, they need to be fixed first before this can be merged.

**Comment by haouarihk** [comment]
i fixed the issue with decode
#812

**Comment by marco-c** [comment]
Merged in #813, thank you!

**Pull request closed without merging**